### ISSUE TITLE
other file formats and data models for annotated arrays

### ISSUE THREAD

**Issue by neuromusic** [ivirshup/sc-interchange#5]
In your landscaping, it might be worth calling out other file formats for annotated arrays. Building a schema on top of these standards could facilitate interoperability.

- netCDF (R, Python, C++, Java, others)
- zarr (Python)
- xarray (Python)
- tile.db (C++, Python, others)

**Comment by ivirshup** [comment]
Yup, a few of these had come up in previous discussion with @LTLA. The idea of Arrow/ Parquet had also come up. 

I like the idea of xarray/ netCDF for the existing infrastructure, though it's unclear to me what the status is of sparse data for netCDF. There has been a little discussion of this on the xarray sparse issue (https://github.com/pydata/xarray/issues/1375).
